### PR TITLE
Fix 32-bytes signature return by Ledger module

### PR DESCRIPTION
**Pull request by TriPhan96**

### CODE REVIEW

**Review by rg911 — CHANGES_REQUESTED** [review]
This still doesn't look right to me. 

For consigning it is the `transactionHash` get cosigned, see the line below, the original Nano wallet option passes `cosignDTO.h` for consigning, that `.h` is the hash of the signed aggregate transaction

**Comment by TriPhan96** [comment]
Updated code

File: src/app/modules/ledger/hw-app-symbol.js [hunk 1]
```diff
@@ -137,6 +137,31 @@ export default class SymbolLedger {
...
+     *
+     * @param path a path in BIP 44 format
+     * @param cosignatureTransaction a cosinature transaction needs to be signed
+     * @param signerPublicKey the public key of signer
+     * @param isOptinSymbolWallet if Opt-in Symbol wallet uses curve Secp256K1 else uses curve Ed25519
+     * @return a Signed Cosignature Transaction which is signed by account at path on Ledger
+     */
+    async signCosignatureTransaction(path, cosignatureTransaction, signerPublicKey, isOptinSymbolWallet) {
+        const rawPayload = cosignatureTransaction.serialize();
+        const signingBytes = cosignatureTransaction.transactionInfo.hash + rawPayload.slice(216);
```
Comment (rg911) [inline src/app/modules/ledger/hw-app-symbol.js:151]: what's the cosignatureTransaction? 
how does this get calculated `cosignatureTransaction.transactionInfo.hash + rawPayload.slice(216);` ?

File: src/app/modules/ledger/optin/cosigOptinDTOLedger.js [hunk 2]
```diff
@@ -44,15 +44,15 @@ exports.CosigOptinDTOLedger = CosigOptinDTOLedger;
...
+CosigOptinDTOLedger.createLedger = async (cosigner, cosignerAccountPath, convertDTO, multisigDestination, network) => {
     const transaction = symbol_sdk_1.TransactionMapping.createFromPayload(convertDTO.p);
     if (transaction instanceof symbol_sdk_1.AggregateTransaction) {
         const cosignatureTransaction = symbol_sdk_1.CosignatureTransaction.create(transaction);
         let signature;
         const ledgerService = new LedgerService();
         if (cosigner.privateKey === undefined) {
-            const result = await ledgerService.signSymbolTransaction(cosignerAcountPath, cosignatureTransaction.transactionToCosign, constants_1.OptinConstants[network].CATAPULT_GENERATION_HASH, cosigner.publicKey);
-            signature = result.hash;
+            const result = await ledgerService.signSymbolAggregateTransaction(cosignerAccountPath, cosignatureTransaction.transactionToCosign, constants_1.OptinConstants[network].CATAPULT_GENERATION_HASH, cosigner.publicKey);
```
Comment (rg911) [inline src/app/modules/ledger/optin/cosigOptinDTOLedger.js:54]: should this just be the `convertDTO.h` ?
why pass the whole aggregate transaction for consigning?

**Pull request closed without merging**